index.html

```diff
@@ -249,6 +269,9 @@ <h3>Contact me</h3>
         <textarea name="user_message" id="msg" maxlength="500" rows="7" placeholder="write your message here."></textarea>
         <button type="submit" class="get-in-touch">Get in touch</button>
```

Comment:
Please stick to the design and align your Get in touch button to the middle for the desktop version.But notice that on the mobile version it is aligned left 👍
<img width="1193" alt="Screenshot 2023-01-26 at 2 03 24 AM" src="https://user-images.githubusercontent.com/25887253/214714024-ad30af38-61ef-4783-9817-39e1d4bb71a9.png">